cluster/selfplay/player_wrapper.sh

```diff
@@ -23,8 +23,8 @@ echo bucket: $BUCKET_NAME
 echo board_size: $BOARD_SIZE
 
 python3 rl_loop.py selfplay \
-  --resign-threshold=0.99 \
-  --readouts=600
+  --resign-threshold=0.88 \
```

Comment:
yuck :D

Reply:
if you've got a better approach i'm all ears ;P